src/EventStore.NETCore.Compatibility/EventStore.NETCore.Compatibility.csproj

```diff
@@ -1,10 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net5.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
     <Platforms>x64;ARM64</Platforms>
     <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="System.ServiceModel.Http" Version="4.7.0" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="6.0.1" />
```

Comment:
was `System.Security.Cryptography.Xml` added by mistake? the project builds without it (on linux)

Reply:
it's a redirect to make the vulnerability check happy

Reply:
ok :+1: